content/docs/setup/kubernetes/multicluster-install/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Multicluster Install
+description: Span Istio mesh across multiple clusters.
+weight: 60
+type: section-index
+keywords: [kubernetes,multicluster]
+---
```

Comment:
please add the missing redirect.  I'm not clear how you get one without renaming the top level directory otherwise circular redirects will occur.